conf/core/ConfigOptions.asciidoc

```diff
@@ -4820,6 +4958,14 @@ useful for debugging.
 When true, writers will include debug information (hoot:* tags; e.g. status). Not honored by all
 writers.
 
+=== writer.include.matched.by.tag
+
```

Comment:
Added this to help make sure the generic geometry scripts weren't trying to steal matches from the better equipped custom routines. Decided to make it configurable outside of `writer.include.debug.tags` just due to the shear amount of test output changes required to default it with that option. Given the size of this PR and the amount of time I've already spent on it, I'm skipping that gargantuan task for the time being.